test/functional/tests/misc/test_device_capabilities.py

```diff
@@ -0,0 +1,220 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+import math
+import pytest
+import os
+from api.cas import casadm, cli_messages
+from core.test_run import TestRun
+from storage_devices.disk import DiskTypeSet, DiskType, DiskTypeLowerThan
+from storage_devices.partition import Partition
+from test_tools import disk_utils, fs_utils
+from test_utils import os_utils
+from test_utils.output import CmdException
+from test_utils.size import Size, Unit
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.require_plugin("scsi_debug")
+def test_device_capabilities():
+    with TestRun.step("Prepare devices. Create disk for core using scsi_debug module."):
+        cache_device = TestRun.disks['cache']
+        core_device = create_scsi_debug_device(512, 4)
+    with TestRun.group("SCSI-debug module | dev_size_mb = 1024 | logical_block_size = 512 | "
+                       "max_sectors_kb = 1024"):
+        with TestRun.step("Start cache and add prepared core device as core."):
+            cache, core, error_output = prepare_cas_device(cache_device, core_device)
+        with TestRun.step("Compare capabilities for CAS device, cache and core "
+                          "(or check proper error if logical sector mismatch occurs)."):
+            compare_capabilities(cache_device, core_device, cache, core, error_output)
+        with TestRun.step("Recreate CAS device with switched cache and core devices."):
+            cache, core, error_output = prepare_cas_device(core_device, cache_device)
+        with TestRun.step("Compare capabilities for CAS device, cache and core "
+                          "(or check proper error if logical sector mismatch occurs)."):
+            compare_capabilities(core_device, cache_device, cache, core, error_output)
```

Comment:
How about adding empty lines between test groups? I think it would increase readability